project_gtd/tests/test_project_gtd.py

```diff
@@ -8,6 +8,9 @@
 class TestProjectGtd(TransactionCase):
     def setUp(self, *args, **kwargs):
```

Comment:
`args` and `kwargs` are not needed: could you remove them?

Reply:
ok, I'll do that. thanks